public/css/landing/style.css

```diff
@@ -3061,6 +3061,14 @@
     gap: 30px;
     padding: 40px 20px;
   }
+  .call-to-action-content-action{
+    gap:0;
+    justify-content: space-between;
+  }
+  .call-to-action-content-action-copy-command-button{
+    max-width:100px ;
+  }
+  
```

Comment:
You don't need to do this much for the fix, and also this fix is not ideal for below reasons:
* It affects the design of the website (leads to uneven space b/w 'Copy' and 'Get Started' buttons for the pixel range).
* The fix only works for a particular pixel range, while the issue still exists beyond that range.

It actually only needs a very simple one-liner fix. The root problem here is not that there is too much space b/w the two buttons but the wrapping of the text inside the "Get Started" button (if you try to enable/disable the `gap` property, you'll notice that the size of the "Get Started" button isn't changing but still the text is wrapping vertically which leads to this UI issue). This issue can simply be fixed by disabling the wrapping of text in the "Get Started" button.

Reply:
this text wrapping is happening bcz the command part has 70% width and have fix gap between the command part and getting started and which lead to the text wrapping . And if we only remove the text wrapping only it still hide the cmd in the sections which we want to show .

Reply:
The hidden command is a non-issue to me, it happens on a small range of width and you can anyway copy the command and use it, or go to the top of the page to see it fully, or just click on the "Get Started" button.

And as another option, we can maybe add a tooltip to the text, so that users can read the full command by hovering over it.

Reply:
Yeah, I think adding tooltip would be good.

Reply:
ok then should i create an separate and PR  issue for that or i comment that too in this request ,

Reply:
You can do that in this PR itself.

Reply:
ok i will update you once i will do this

Reply:
@garg3133 i added the tooltip have a look at it will work after 1024px only and , let me know the styling changes you want to have to meet the site design

Reply:
@mukeshblackhat You didn't have to do this much to add a tooltip. Tooltip is an HTML feature and can be set by just adding a `title` attribute to the element.

The tooltip you added anyway looks a bit off (it is coming over the actual command text) and it's fine to have the tooltip for all screen-widths.

Reply:
@garg3133 i have done the changes you asked about .